crates/ruff_python_resolver/src/native_module.rs

```diff
@@ -8,7 +8,26 @@ pub(crate) fn is_native_module_file_extension(file_extension: &OsStr) -> bool {
     file_extension == "so" || file_extension == "pyd" || file_extension == "dylib"
 }
 
-/// Returns `true` if the given file name is that of a native module.
-pub(crate) fn is_native_module_file_name(_module_name: &Path, _file_name: &Path) -> bool {
-    todo!()
```

Comment:
Should we enable https://rust-lang.github.io/rust-clippy/master/index.html#/todo and https://rust-lang.github.io/rust-clippy/master/index.html#/dbg_macro for CI only?